packages/components/src/components/tag/tag.stories.ts

```diff
@@ -29,173 +32,90 @@ export const Default = {
 };
 
 /**
- * The tag in all possible combinations of `selected` and `size`.
+ * Use the `size` attribute to change the tag size. The default size is `lg`.
  */
 
-export const selectedAndSize = {
-  name: 'Selected × Size',
-  parameters: { controls: { exclude: ['size', 'selected'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'selected', values: ['false', 'true'] },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+export const Size = {
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-tag size="lg">Large</sd-tag>
+      <sd-tag size="sm">Small</sd-tag>
+    </div>
+  `
 };
 
 /**
- * The tag in all possible combinations of `removable` and `size`.
+ * Use the `selected` attribute to enable the selected state.
  */
 
-export const removableAndSize = {
-  name: 'Removable × Size',
-  parameters: { controls: { exclude: ['size', 'removable'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'removable', values: ['false', 'true'] },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+export const Selected = {
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-tag selected>Selected</sd-tag>
+    </div>
+  `
 };
 
 /**
- * Use the `disabled` attribute to disable a tag. Clicks will be suppressed until the disabled state is removed.
- *
- * __Hint:__ If the href attribute is set i.e. the tag is rendered as a link (`<a>`),
- * the disabled attribute is ignored, as links may not be disabled.
- * To disable the tag in this case the href attribute has to be removed as well.
+ * Use the `removable` attribute to include the removability indicator and make the tag removable from the UI.
```

Comment:
```suggestion
 * Use the `removable` attribute to include the removability indicator.
```

Let's make it more like this, so removability is not bound to the indicator.